packages/desktop-client/src/components/manager/BudgetList.tsx

```diff
@@ -444,23 +443,24 @@ export function BudgetList({ showHeader = true, quickSwitchMode = false }) {
           }}
         >
           <Button
-            type="bare"
+            variant="bare"
+            aria-label="Import file"
             style={{
               ...narrowButtonStyle,
               marginLeft: 10,
               color: theme.pageTextLight,
             }}
-            onClick={e => {
-              e.preventDefault();
+            onPress={() => {
               dispatch(pushModal('import'));
             }}
           >
             Import file
           </Button>
 
           <Button
-            type="primary"
-            onClick={() => onCreate()}
+            variant="primary"
+            aria-label="Create new file"
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
```